Doc/library/datetime.rst

```diff
@@ -590,7 +590,7 @@ Instance methods:
 
 .. method:: date.isocalendar()
 
-   Return a 3-tuple, (ISO year, ISO week number, ISO weekday).
+   Return a :class:`IsoCalendarDate`, (ISO year, ISO week number, ISO weekday).
```

Comment:
@pganssle 

> The pure-Python version defines a simple tuple sub-class with attribute access to the 
>  items, closely mimicking the structseq used by the C version. 

`:term:named tuple` is the right choice?
We are actually mimicking structseq aren't we?

Reply:
Unfortunately, the Glossary has entries for both "named tuple" and "struct sequence", and it's clear as mud what the difference is intended to be.  At this point I'd be in favor of getting rid of the phrase "struct sequence" entirely everywhere in the docs, using only "named tuple" instead.  But that's outside the scope of this PR. 

The abbreviated name "structseq" doesn't mean anything to anyone outside CPython developers - it's an internal name used in CPython's C implementation code.

So, bottom line:  Paul's "`named tuple` object" looks best to me for now, although it will become more obviously clearer later, when the current "struct sequence"/"named tuple" confusion is purged from the docs.